lib/vanagon/platform/windows.rb

```diff
@@ -82,6 +85,8 @@ def generate_msi_package(project)
         # Actual array of commands to be written to the Makefile
         ["mkdir -p output/#{target_dir}",
         "mkdir -p $(tempdir)/#{project.name}/staging",
+        "mkdir -p $(tempdir)/wix",
+        "#{@copy} *.wxs $(tempdir)/wix",
```

Comment:
yeah, I'd suggest not worrying about that specific case until we implement it. We might think we're going to solve it one way, and then end up solving it a completely different way